File: frontend/src/constants/orderEnums.ts
export const ORDER_TYPES = ["TAKEAWAY", "DELIVERY"] as const;
export const PAYMENT_TYPES = ["CASH", "CARD"] as const;
export const ORDER_STATUS = [
  "PENDING",
  "CONFIRMED",
  "PREPARING",
  "OUT_FOR_DELIVERY",
  "COMPLETED",
  "CANCELLED",
] as const;

File: backend/enums/order_enums.py
from enum import Enum

class OrderType(Enum):
    TAKEAWAY = "TAKEAWAY"
    DELIVERY = "DELIVERY"
    DINE_IN = "DINE_IN"

class PaymentType(Enum):
    CASH = "CASH"
    CARD = "CARD"
    ONLINE = "ONLINE"

class OrderStatus(Enum):
    PENDING = "PENDING"
    CONFIRMED = "CONFIRMED"
    PREPARING = "PREPARING"
    OUT_FOR_DELIVERY = "OUT_FOR_DELIVERY"
    COMPLETED = "COMPLETED"
    CANCELLED = "CANCELLED"
